### PR TITLE
binder page

### PR DESCRIPTION
**Pull request by choldgraf**
Adds a page with information about the Binder project.

Currently, this doesn't change the navbar because there isn't much room there for a new item. For now, we can just link directly to this page when people have questions about
the Binder project.

### CODE REVIEW

File: binder.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+layout: page_md
+title: The Binder Project
+title_image: assets/logo_binder.svg
+tagline: Reproducible, sharable, interactive computing environments
+permalink: /binder
+---
+
+## What is the Binder Project?
+
+The Binder Project is a community that builds free and open-source tools
```
Comment (betatim) [inline binder.md]: How about something like:

"The Binder project brings the power of customisable and easy to share computing environments to everyone. It allows users to specify custom environments, automates their creation and allows users to share them with a single link. Use-cases involve workshops, scientific workflows and fast sharing of work in teams."

I feel like open-source and free aren't the most important things to clarify here. We are on the Jupyter web site, everything here is open-source and free. I tried to follow the JupyterHub page a bit by trying to explain when/why/who would use Binder. I am imagining that the reader of this first paragraph is someone who is clicking around the website looking for tools that exist, either with a specific use case in mind (do they have something to help me with that?) or just to inform themselves (lets see what they have and what it is good for). This means I think we should focus on use-cases, what pain the user has that we solve and/or what you get when you use binnder.

Comment (choldgraf) [reply binder.md]: I like it - I've updated with your language. I think it's more content-ful now!

**Comment by betatim** [comment]
I've been pondering who will read this page, what they already know and what they will be looking for. Once we know this we can write text for them.

Here is my guess: either familiar with Jupyter-the-notebook or just heard of Jupyter but unfamiliar. Not super informed about what Project Jupyter is and the different options and sub-communities that exist. Not because they don't care but keeping up with open-source politics isn't what they do day to day. They are just using "Jupyter". They are wondering what else Jupyter does out of interest or because they have a concrete problem that they would like to solve and wondering if Jupyter "already has something for this".

As a result I think we should describe what problems Project Binder solves in a way that people who have this problem can recognise themselves. We can't make an exhaustive list because that list is super long. My feeling is that the three most frequent uses of mybinder.org are:
* workshops and tutorials (Scipy and Pycon workshops. "do you also love spending the first half of your workshop sorting out how to install stuff?")
* documentation and examples (spacy, bokeh, scikit-learn, etc)
* education (lecture courses and books)

I would lead with these (or other) three examples of when you want to use Binder. It is important to include the badge in each example or other link that is "try it now". Binder is all about making trying stuff easy and low risk so we need to use that in our page :)

Then a blurb about what Project Binder (PB) is (we already have that in the current diff).

Then a list of the things PB does, with their GitHub repo descriptions as description (maybe we need to update them a bit), link to the docs, link to "how to get started", link to the repo
* BinderHub
* repo2docker
* mybinder.org
* binder example gallery(?)

(Put like that it seems like we don't do that much...)

Then a short block on "where to find out more, come chat to us, ask on the forum, get involved with contributing to the individual repositories. Thanks to GKE and OVH for sponsoring mybinder.org"

---

What do you think  about that as a rough plan? (Also how did it take till now for me to realise that Project Binder + Jupyter can be shortened to PB+J that mos awesome of sandwich toppings?)

**Comment by choldgraf** [comment]
Finally looping back on this one too :-)

Thanks @betatim for the comments - I updated the content and pushed a new commit w/ some extra info, what do you think?

https://5d8e72bd1755f90008c7b093--dazzling-gates-2d8e39.netlify.com/binder

One thing to note in terms of "who will land on this page?". Right now, it isn't linked from anywhere in the Jupyter documentation. The link just "exists" and we could link to it externally. So I doubt (in its current form) this page would just be stumbled upon by curious Jupyter people, but I might for example include a link to this page when I give talks about Binder etc.

**Comment by betatim** [comment]
Not linking it from the navbar means the pressure is lower to get it right first time. I like the idea of linking it from other places and the fact that it is on jupyter.org will give it some credibility.

How do we move forward getting this merged? Do we ping someone? Mark it as ready for review?

**Comment by choldgraf** [comment]
good question - @tgeorgeux could you take a look and merge if all looks OK? This PR doesn't change the front page of jupyter.org, it only makes this page available if somebody directly links to it (which I'd like to do occasionally in talks etc)

File: binder.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+layout: page_md
+title: The Binder Project
+title_image: assets/logo_binder.svg
+tagline: Reproducible, sharable, interactive computing environments
+permalink: /binder
+---
+
+## What is the Binder Project?
+
+The Binder project brings the power of customisable and easy to share computing
```
Comment (tgeorgeux) [inline binder.md]: Is this spelling correct? I've only "seen customizable", is it a British thing?

File: binder.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+automates their creation, and allows users to these environments with a single link.
+Use-cases involve workshops, scientific workflows and fast sharing of work in teams.
+
+The Binder Project builds tools that reward best-practices in reproducible data
+science by piggy-backing off of community-developed standards for
+reproducibility. If repositories follow these best-practices and are hosted in
+an online repository, then Binder will build the environment needed to run the
+code and provide a link that allows others to interact with the results and
+reproduce analyses.
+
```
Comment (tgeorgeux) [inline binder.md]: Overall this intro is a bit verbose. I don't know exactly who your target audience is, but as a general rule I would not assume casual observers would read this much text.

Comment (tgeorgeux) [reply binder.md]: So I'm also offering something helpful here's a shortened version:
"The Binder project offers an easy place to share computing environments to everyone. It allows users to specify custom environments and share them with a single link. Use-cases involve workshops, scientific workflows and streamline sharing among teams."

The Binder Project builds tools that reward best-practices in reproducible data
science by utilizing community-developed standards for
reproducibility. When repositories follow these best-practices and are hosted in
an online repository, then Binder automatically builds a linkable environment anybody can access."

**Comment by tgeorgeux** [comment]
Also of note, I don't have merge rights on this repo. :)

File: binder.md (hunk 2 above)
Comment (betatim) [reply binder.md]: Yes customisable is the UK spelling

File: binder.md (hunk 3 above)
Comment (choldgraf) [reply binder.md]: just added in this language verbatim - nice!

**Comment by choldgraf** [comment]
Thanks for the comments @tgeorgeux - I *think* should should be able to merge, since GitHub blocks a merge until someone with merge rights approves the PR, and your approval has unlocked the ability to merge for me, so I assume you have merge rights?

If not, perhaps I can merge this away and then link to it from the docs.mybinder.org page?

**Comment by tgeorgeux** [comment]
My permissions are at a weird stage where I am an 'approving reviewer' but still can't merge unless somebody ELSE who is an approver approves. Regardless I don't really see any reason not to merge if you've got the button!

**Comment by choldgraf** [comment]
okie dokie - I'll merge this since it's not changing the front page / nav bar etc at all, thanks for the review and improvements!

**Pull request merged**